azure-posix.yml

```diff
@@ -82,6 +82,12 @@ jobs:
         displayName: Install wheel and test
         condition: ne(variables['PLAT'], 'arm64')
 
+      - bash: |
+          set -xe
+          for file in `find ./wheelhouse -type f -name '*11_0_arm64*.whl'`; do mv -v "$file" "${file/11_0_arm64/12_0_arm64}"; done
+        displayName: "Rename MacOS arm64 wheels for version 12.0 minimum"
+        condition: eq(variables['PLAT'], 'arm64')
```

Comment:
this is slight overkill since there's only one wheel per matrix entry, but it does seem to work locally and the paradigm was conveniently available from a stackoverflow snippet.. let's see what CI says..